scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -3002,12 +3003,26 @@ def setup(params):
       formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
   # Sets environment variables for MKL
+  # If OS ENV variables are overridden, a warning msg is printed.
   if params.mkl:
+    if '-kmp_blocktime' not in str(sys.argv) and 'KMP_BLOCKTIME' in os.environ:
+      tf.logging.warn("OS ENV variable KMP_BLOCKTIME is ignored and " +
+                    " script default is used. Use --kmp_blocktime to override.")
```

Comment:
Line up the starting quote with the starting quote on the previous line, and same for for other tf.logging.warn lines below.